apps/condo/domains/contact/hooks/useImporterFunctions.tsx

```diff
@@ -1,12 +1,18 @@
+import { useAddressApi } from '@condo/domains/common/components/AddressApi'
 import { Columns, ObjectCreator, RowNormalizer, RowValidator } from '@condo/domains/common/utils/importer'
-import { useOrganization } from '@condo/next/organization'
+import { Contact, ContactRole } from '@condo/domains/contact/utils/clientSchema'
+import {
+    APARTMENT_UNIT_TYPE,
+    COMMERCIAL_UNIT_TYPE,
+    FLAT_UNIT_TYPE,
+    PARKING_UNIT_TYPE,
+    WAREHOUSE_UNIT_TYPE,
+} from '@condo/domains/property/constants/common'
+import { searchContacts, searchProperty } from '@condo/domains/ticket/utils/clientSchema/search'
 import { useApolloClient } from '@condo/next/apollo'
-import { useAddressApi } from '@condo/domains/common/components/AddressApi'
-import get from 'lodash/get'
-import { Contact } from '../utils/clientSchema'
-import { searchProperty, searchContacts } from '@condo/domains/ticket/utils/clientSchema/search'
 import { useIntl } from '@condo/next/intl'
-import { FLAT_UNIT_TYPE, APARTMENT_UNIT_TYPE, WAREHOUSE_UNIT_TYPE, PARKING_UNIT_TYPE, COMMERCIAL_UNIT_TYPE } from '@condo/domains/property/constants/common'
+import { useOrganization } from '@condo/next/organization'
+import get from 'lodash/get'
```

Comment:
Best to reorder imports: 

1. external imports (lodash, next)
2. internal imports from `@packages` 
3. internal imports from `@common`
4. other imports

Reply:
Imports sorted alphabetically `ctrl` + `alt` + `o`